vtex-apps/listrak-pixel
# [LISTRAK-10 and LISTRAK-7] Fixed product import payload, Swapped color and size specifications

Listrak app was sending a single product object instead of an array of products, causing `createProduct` to fail. Here's the Listrak API documentation: https://api.listrak.com/data#operation/Product_PostProductCollection

App was also sending product with color and size specifications swapped.

Testing:
1. Update a product or SKU
2. Check Splunk for `ProductHandler-createProductResponse` logs. Verify log data for `listrakProduct` color and size values. (Query: `index=io_vtex_logs vtex.listrak-pixel* "data.message"="ProductHandler-createProductResponse"`)
